examples/react/3.two-list-conversation/README.md

```diff
@@ -0,0 +1,208 @@
+# Examples: 3.two-list-conversation
+
+The code about 'typing' example, here show you the code necessary to implement this behaviors.
```

Comment:
The code about 'typing' example, here show you the code necessary to implement this behaviors. =>
The code about 'typing' example, here we show you the code necessary to implement this behavior.